In [56]:
include("../math-functions/continued-fractions.jl")

cf2num (generic function with 1 method)

In [84]:
cf = []
for i in 1:10
    push!(cf,i)
end

n = cf2num(cf)

for i in 1:16
    println(trunc(n,i)," : ",rationalize(trunc(n,i)))
end

1.4 : 7//5
1.43 : 143//100
1.433 : 1433//1000
1.4331 : 14331//10000
1.43312 : 8957//6250
1.433127 : 1433127//1000000
1.4331274 : 7165637//5000000
1.43312742 : 71656371//50000000
1.433127426 : 117788114//82189561
1.4331274267 : 91092649//63562142
1.43312742672 : 84103341//58685180
1.433127426722 : 100168377//69894955
1.4331274267223 : 54645712//38130393
1.43312742672231 : 53164142//37096591
1.433127426722315 : 7489051//5225670
1.433127426722315 : 7489051//5225670


#### $$ 1.433127\ldots = 1 + \cfrac{1}{2+\cfrac{1}{3+\cfrac{1}{4+\cfrac{1}{5+\cfrac{1}{6+\cfrac{1}{7+\dots}}}}}} $$